test/topology_experimental_raft/test_topology_ops.py

```diff
@@ -24,7 +24,13 @@
 @pytest.mark.parametrize("tablets_enabled", ["true", "false"])
 async def test_topology_ops(request, manager: ManagerClient, tablets_enabled: bool):
     """Test basic topology operations using the topology coordinator."""
-    cfg = {'experimental_features' : ['tablets']} if tablets_enabled else {}
+    rf = 3
+    if tablets_enabled:
```

Comment:
Isn't `tablets_enabled` is a non empty string here which is always `True`?

Reply:
Yes, and we have multiple (5 if I'm not wrong) tests broken in the same way. The first bug served as a bad example that caused the other bugs. We need to fix these tests and check if they pass.

The bigger problem with this particular test is that before this PR we didn't send writes if tablets were enabled. But they were always enabled, so we never sent writes. The test was basically disabled. This is very bad for such a basic and important test.

Hopefully, the situation is not that bad. IIRC, I verified that this test was not flaky before incorrectly parameterizing it with tablets around 3 month ago. So, I hope that "missing IP mapping" is the only regression.

@kbr-scylla

Reply:
> Yes, and we have multiple (5 if I'm not wrong) tests broken in the same way. The first bug served as a bad example that caused the other bugs. We need to fix these tests and check if they pass.

Only one test was broken -- `test_default_tombstone_gc`. I opened #18888.

I'll be investigating failures in this test after re-enabling writes. This PR will have to wait until all failures are fixed.

Reply:
The failures are because the test is always running with tablets (https://github.com/scylladb/scylladb/pull/18707#issuecomment-2132232745) and it is a known problem with tablets https://github.com/scylladb/scylladb/issues/18843

Reply:
but my observation is quite the opposite , it fails with write load enabled and with tablets disabled. see #18932

Reply:
You need to move the test from topology_experimental_raft since this suit enables tablets by default. See https://github.com/scylladb/scylladb/pull/18707#issuecomment-2132232745.

Reply:
I fixed it in #18900 by changing the config.